website/src/components/PreviousExecutionResponsiveTablet/PreviousExeResTablet.jsx

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {useState} from 'react';
+import moment from 'moment';
+
+import '../PreviousExecutionResponsiveTablet/previousExeResTablet.css'
+
+const PreviousExeResTablet = ({data, className}) => {
+
+    const getStatusClass = (status) => {
+        switch (status) {
+            case 'finished':
+              return 'finished';
+            case 'failed':
+              return 'failed';
+            case 'started':
+              return 'started';
+            default:
+              return 'default';
+          }
+    }
```

Comment:
I see `getStatusClass` is being defined in multiple places. It would be wise to create some sort of utils file and gather all the utility functions like this one instead of re-declaring the same code in multiple places.